docs/user/access-control.md

```diff
@@ -40,12 +40,25 @@ Kubeapps.
 
 #### Read access to Applications within a namespace
 
-In order to list and view Applications in a namespace, apply the `kubeapps-applications-read`
-ClusterRole in the desired namespace. Note that the `kubeapps-applications-read` role provides
-read access to **all** the resources of a namespace so apply it carefully. In case you want
-to limit the access create a custom cluster role or use one of the [default ones](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles).
+In order to list and view Applications in a namespace, first we will create a `ClusterRole` with read-access to **all** the possible resources. In case you want
+to limit this access, create a custom cluster role or use one of the [default ones](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles). Then we will bind that cluster role to our service account.
 
 ```
+kubectl apply -f - << EOF
```

Comment:
Maybe for a first version it might make sense to store these yaml in the repo so it can be easier installed via kubectl create -f 'http://github.com/foobar.yaml'

Reply:
I like having the role definition here because it's more verbose and users won't be applying them blindly. Also we'd avoid possible broken links if we move/delete those files from the repo.

In any case, if you think it's a price worth to pay for the simplicity it provides I am fine as well.

Reply:
rethinking about this, having a file is more windows-friendly (I don't think the `<< EOF` works there)